docs/docs/widgets/pagination.md

```diff
@@ -0,0 +1,71 @@
+---
+id: pagination
+title: Pagination
+---
+# Pagination
+
+Pagination enables the user to select a specific page from a range of pages.
+
+:::tip
+It is used seperate content into discrete pages.
+:::
+
+<div style={{textAlign: 'center'}}>
+
+![ToolJet - Widget Reference - Code editor](/img/widgets/pagination/pagination.png)
+
+</div>
+
+### Properties
+
+#### Number of pages
+
+You can use this to predefined the total number of pages. It is calculated by dividing the length of the data array that will be passed, by the data limit which is the number of posts we will show on each page. 
+
+#### Default page index
+It is used to set and display the default page index when the app initially loads. You can also put a conditional logic to set its value as per your use case.
+
+### Event 
+
+#### On Page Change
+
+This event is triggered whenever the user switches to another page index. You can explore various actions associated with this event as per app logic.
+
+### General
+#### Tooltip
+
+A Tooltip is often used to specify extra information about something when the user hovers the 
+mouse pointer over the widget.
+
+Under the <b>General</b> accordion, you can set the value in the string format. 
+Now hovering over the widget will display the string as the tooltip.
+
+<div style={{textAlign: 'center'}}>
+
+![ToolJet - widget- button](/img/tooltip.png)
+
+</div>
```

Comment:
Replace with `<img>`

Reply:
Updating <img> tag here with outer div, as this image gets left aligned if div not provided.